examples/buttons/modules/buttons.js

```diff
@@ -0,0 +1,23 @@
+var buttonHandler = {};
+
+window.onPress = function onPress(type, callback) {
+  if (['back', 'up', 'select', 'down'].indexOf(type) >= 0) {
```

Comment:
you have this nice mapping here, but down in `addEventListener()` you do an `if … else` cascade again.

Looking at #73 and our C-API I see that clients want to distinguish between short and long presses. How would you do that?